src/main/java/org/baeldung/security/LoggedUsers.java

```diff
@@ -0,0 +1,32 @@
+package org.baeldung.security;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.baeldung.web.dto.LoggedUserDto;
```

Comment:
I could not find this file when cloning the repo. Did you forget to check in LoggedUserDto.java?